cli/src/main/kotlin/com/malinskiy/marathon/cli/config/deserialize/GraphiteConfigurationDeserializer.kt

```diff
@@ -0,0 +1,29 @@
+package com.malinskiy.marathon.cli.config.deserialize
+
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+import com.malinskiy.marathon.exceptions.ConfigurationException
+import com.malinskiy.marathon.execution.AnalyticsConfiguration
+
+class GraphiteConfigurationDeserializer
+    : StdDeserializer<AnalyticsConfiguration.GraphiteConfiguration>(
+    AnalyticsConfiguration.GraphiteConfiguration::class.java
+) {
+    override fun deserialize(p: JsonParser?, ctxt: DeserializationContext?): AnalyticsConfiguration.GraphiteConfiguration {
+        val node: JsonNode? = p?.codec?.readTree(p)
+
+        val host = node?.get("host")?.asText()
+        val portString = node?.get("port")?.asText()
+        val prefix = node?.get("prefix")?.asText()
+
+        val port = portString?.toIntOrNull()
+
+        if (host == null) throw ConfigurationException("GraphiteConfigurationDeserializer: host should be specified")
```

Comment:
To be honest, I used `InfluxDbConfigurationDeserializer` as an example and created a similar implementation for Graphite. But yes, it displays detailed error logs when the configuration is incorrect. Do you think this should be handled differently?

Reply:
No, that's fine, leave it as is. 
We're thinking of moving to something else in terms of configuration parsing so that we get proper error messages out-of-the-box instead of writing custom deserialisers. The whole point was to define data model and let the parse handle everything, but json/yaml packages lack proper error handling for our particular case.